docs/flink-k8s/1-deployment.md

```diff
@@ -14,9 +14,9 @@ At now, one StreamPark only supports one Kubernetes cluster.You can submit [Fear
 
 ## Environments requirement
 
-Additional operating environment to run StreamPark Flink-K8s is as below:
+Additional operating environment to run StreamPark Flink-Kubernetes is as below:
 * Kubernetes
-* Maven（StreamPark runNode）
+* Apache Maven（StreamPark runNode）
```

Comment:
This is another pattern that we wrongly use full-width characters in English content. We should use half-width characters, that is:

```
* Apache Maven (StreamPark runNode)
```